src/platform/silabs/wifi/WifiInterface.h

```diff
@@ -410,6 +412,20 @@ class WifiInterface
      */
     virtual void CancelScanNetworks() = 0;
 
+    using WiFiBandEnum = app::Clusters::NetworkCommissioning::WiFiBandEnum;
+    /**
+     *  @brief Provide all the frequency bands supported by the Wi-Fi interface.
+     *
+     *  The default implementation returns the 2.4 GHz band support.
+     *
+     *  @return a bitmask of supported Wi-Fi bands where each bit is associated with a WiFiBandEnum value.
+     */
+    virtual uint32_t _GetSupportedWiFiBandsMask() const
+    {
+        // Default to 2.4G support only
+        return static_cast<uint32_t>(1UL << chip::to_underlying(WiFiBandEnum::k2g4));
```

Comment:
Do we know the bitmap expected mapping? Where is this defined?

Reply:
this is copied from the MATTER codebase; this is the default function provided for the same `GetSupportedWiFiBandsMask` function. The details are added as comment on that function documentation.